src/swapnaWJun20/PalindromeString.java

```diff
@@ -0,0 +1,24 @@
+package swapnaWJun20;
+
+public class PalindromeString {
+
+	void findPalindromeNumber(String input) {
+		String input1 = input;
```

Comment:
Here you don't need an extra variable to store the given string. the other variable reverse will work.

Reply:
Yes Anup. I got the point here. rectified the same